### ISSUE TITLE
Allow module to be used without replacing rand

### ISSUE THREAD

**Issue by colinnewell** [badalex/Rand-Urandom#1]
I really like what you've done with the rand function, but I want to import it as a different name rather than clobber the existing rand function.  Perhaps as something like `secure_rand` instead.  Would a pull request allowing that as an option be welcome?

**Comment by badalex** [comment]
Yes, of course. I'd probably go for something more like urand() or rand_rand(), urand_rand() etc. I don't know why, but "secure_rand()" tends to scare me a bit, this module makes no security guarantees, only that it will use the random/entropy syscall or /dev/urandom. Both of which should be "secure".

BTW I actually have a patch or two locally that I really should push, mostly to improve performance by introducing a small, but configurable buffer for the syscall and reads from /dev/urandom.

Thanks,
-Alex
